src/store/NotificationStore.js

```diff
@@ -12,13 +14,27 @@ export default class NotificationStore {
 
     @observable messages = [];
 
-    @action.bound notify(text, type = NotificationStore.TYPE_WARNING) {
+    // Duration is in seconds; set to < 0 if you want the notification to remain indefinitely
+    @action.bound notify(text, type = NotificationStore.TYPE_WARNING, duration = 10) {
```

Comment:
@ashkanx I've set the notifications to be removed after 10 seconds.

Reply:
How about I change the symbol in just few seconds? do we remove the notification on symbol change?